_pages/docs/testing-with-xquery.md

```diff
@@ -5,17 +5,233 @@ lead_text: ''
 permalink: /docs/testing-with-xquery/
 ---
 
-# Testing With XQuery
+## Testing XQuery With XQuery
 
-*TODO*
+### Scenario 1 - Test assert all exist with assert-equal for success
+```
+xquery version "1.0-ml";
 
-## Testing XQuery With XQuery
+import module namespace test="http://marklogic.com/roxy/test-helper" at "/test/test-helper.xqy";
```

Comment:
This is the wrong namespace for test-helper.xqy.  It's now `http://marklogic.com/test`.